app/views/users/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<% provide(:title, 'User profile') %>
+<p id="notice"><%= notice %></p>
+
+<p>
+  <strong>Name:</strong>
+  <%= @user.name %>
+</p>
+
+<p>
+  <strong>Email:</strong>
+  <%= @user.email %>
+</p>
+
+<%= link_to 'Edit', edit_user_path(@user) %> |
+<%= link_to 'Back', users_path %>
+
+<p>Current user's id: <%= @current_user.id %></p>
```

Comment:
why you need @current_user instance variable
use devise, it has helper current_user